code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -79,11 +84,6 @@
 			visible_message("<span class ='danger'>[src] smashes into [hit_atom]!</span>", "<span class ='alertalien'>[src] smashes into [hit_atom]!</span>")
 			Paralyze(40, 1, 1)
 
-		if(leaping)
```

Comment:
does leaping not get disabled when you hit now?

Reply:
Good catch.

Its undocumented in the tg changelog, so was it an unintentional change from tg? Probably something I'm missing without the mobility update.

I'll make a note to revisit it when update_mobility is no longer an issue